packages/palette-tokens/src/themes/v3Dark.tsx

```diff
@@ -14,46 +14,46 @@ const COLORS: Colors = {
   /** Suitable for text on black60 and darker */
   black5: "#1a1a1a",
   /** Suitable for text on black60 and darker */
-  white100: "#000000",
+  white100: "#121212",
 
   /** Suitable for text on black10 and lighter */
   blue200: "#d3d9fd",
   /** Suitable for text on black10 and lighter */
   blue150: "#a2b1fb",
   /** Suitable for text on black10 and lighter */
-  blue100: "#707eff",
+  blue100: "#0f77ff",
```

Comment:
@MounirDhahri - if other changes were mixed in here, mind updating the PR title and description to better reflect the reasoning here? Not exactly following